source/WindowScroller/WindowScroller.css

```diff
@@ -0,0 +1,3 @@
+.disabled > * {
+  pointer-events: none !important;
```

Comment:
I’m using this instead of inline styles because a custom scroll container may have many children that need `pointer-events: none`, whereas scrolling the window only needs to apply that style to `body`. So, I wasn’t sure whether to add `!important` or not. It ought to work pretty similarly to the way it used to (replacing inline styles), and it is still overrideable (higher-specificity selectors with `!important`, or inline styles with `!important`). On the other hand, I hate it when libraries use `!important`.